samples/Directory.Build.props

```diff
@@ -12,5 +12,26 @@
     <IsPackable>false</IsPackable>
     <MainLibraryPath>$(MSBuildThisFileDirectory)../src/System.Device.Gpio/</MainLibraryPath>
     <IotBindingsLibraryPath>$(MSBuildThisFileDirectory)../src/Iot.Device.Bindings/</IotBindingsLibraryPath>
+    <PreferPackageReference>false</PreferPackageReference>
```

Comment:
NIT: you don't need this. You are not checking anywhere if it is set to false, you are just checking if it is not true, whcih by default that's the case (it is empty by default).